### PR TITLE
add Kakarot zkEVM project

### PR DESCRIPTION
**Pull request by tcoratger**
For more information, see:
- Website: https://www.kakarot.org/
- Doc: https://docs.kakarot.org/

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -51,6 +51,7 @@ Publications using or mentioning Reth.
  - [sp1-reth](https://github.com/succinctlabs/sp1-reth), a zkEVM PoC by [Succint Labs](https://succinct.xyz) demonstrating transaction costs to the order of $0.01 USD atop Reth
  - [scroll-reth](https://x.com/gakonst/status/1788548434393210938), a port of Scroll to Reth
  - [pevm](https://github.com/risechain/pevm), a massively-parallelised EVM implementation by [RISE Labs](https://www.riselabs.xyz) that is explicitly compatible with Reth ([blog post](https://medium.com/@rise_chain/rise-pevm-parallel-evm-bdfc4bc9f38e), [Tweet](https://x.com/gakonst/status/1798165192460976195))
+ - [Kakarot zkEVM](https://github.com/kkrt-labs), a zkEVM built in Cairo (provable by design) for which the [RPC](https://github.com/kkrt-labs/kakarot-rpc) is constructed using Reth.
```
Comment (jmcph4) [inline README.md]:
```suggestion
 - [Kakarot zkEVM](https://github.com/kkrt-labs/kakarot-rpc), a zkEVM built in Cairo (provable by design) for which the RPC component is constructed using Reth
```

**Review by jmcph4 — CHANGES_REQUESTED** [review]
I had no idea Kakarot was using Reth!

**Comment by tcoratger** [comment]
Hey @jmcph4 anything I can do here to merge?

**Comment by jmcph4** [comment]
> Hey @jmcph4 anything I can do here to merge?

Done. Thanks for the contribution!

**Pull request merged**